config.json

```diff
@@ -1466,6 +1466,15 @@
           "unsafe"
         ]
       },
+      {
+        "slug": "secret-handshake",
+        "name": "Secret Handshake",
+        "uuid": "8c044530-9deb-4ff7-a638-98d6c05ebbb8",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4,
+        "topics": []
+      },
```

Comment:
Oh, I thought that https://github.com/exercism/rust/blob/89459bf584f5154e7ec59f8f36b1d603b1a68177/_test/verify_exercise_difficulties.sh#L48-L54 was supposed to ensure that difficulties are sorted, but it got commented out. That is frustrating

Reply:
Okay, since that check is commented out, there is no order requirement. No action needed. This exercise can stay where it is.